scripts/babylon-integration/utils/teardown.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -uo pipefail
+
+source "./common.sh"
+
+# Get the consumer FP address
+KEYRING_DIR=/home/.babylond
+CONSUMER_FP_KEYRING_DIR=$KEYRING_DIR/$CONSUMER_FINALITY_PROVIDER_KEY
```

Comment:
jfyi we will need to change this path after https://linear.app/snapchain/issue/SNA-543/babylon-deployment-move-consumer-fp-keyring-dir is done

Reply:
got it, ill leave it as is for now to avoid code regression from too many changes